packages/genesys-spark/src/utils/get-closest-element.ts

```diff
@@ -0,0 +1,23 @@
+//https://inindca.atlassian.net/browse/COMUI-2673
```

Comment:
Is there a way to not duplicate this helper file in 2 places?

Reply:
I agree with you, this is why I created the COMUI-2673 ticket to follow up on this issue. Do we this this is a blocker for this PR?

Reply:
Ok awesome. I don't think it's a blocker for this PR.